### ISSUE TITLE
Purr Pals: Purrfection - Graphical bug ingame

### ISSUE THREAD

**Issue by JohnGodgames** [citra-emu/citra#1593]
Missing enviroment...

![purp](https://cloud.githubusercontent.com/assets/11453507/14077636/e021b970-f4ed-11e5-8e14-2d2b82475d35.JPG)
![purp 2](https://cloud.githubusercontent.com/assets/11453507/14077637/e2864d02-f4ed-11e5-9929-e1b7cf03f629.JPG)

[LOG](http://pasted.co/00218088)

**Comment by JayFoxRox** [comment]
Might be software blit (does it work with sw-rasterizer?), GS, stencil or depth ( #1621 ? )

**Comment by JohnGodgames** [comment]
It crashed with sw renderer while attempting to ingame

**Comment by JayFoxRox** [comment]
Can be closed once #1740 is merged.

We'll need a new issue about SW Rast crashing.
